### PR TITLE
FEATURE: RAIL-741 Introduce onLoadingFinish to Execute component

### PR DESCRIPTION
**Pull request by BugsBunny338**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/81/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1330/ : FAILURE in 1m 59s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/84/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1333/ : SUCCESS in 3m 10s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/94/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1345/ : FAILURE in 2m 38s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/95/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1346/ : FAILURE in 35s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/96/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1347/ : SUCCESS in 3m 15s

File: src/interfaces/Events.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export interface ILoadingState {
 
 export type OnError = (error: RuntimeError) => void;
 export type OnLoadingChanged = (loadingState: ILoadingState) => void;
+export type OnLoadingFinish = (result: object) => void;
```
Comment (ivan-nejezchleb) [inline src/interfaces/Events.ts]: Don't we have proper type for exec result already?

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/13/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1348/ : SUCCESS in 4m 43s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/553/ : SUCCESS in 12m 19s

**Pull request merged**